### DISCUSSION TITLE
NTSC-RT Ultimate Preset Pack V0.1.1

### DISCUSSION THREAD

**Discussion by smfthe1199** [ntsc-rs/ntsc-rs#157] [General]
(updated and this post made by smfthe1199)

### What's new?

1. New preset creators:
  - @notdigidani
  - @VexorionReal
  - @ThrivertonVHS

2. _"Drone Analog Effects"_ has been added, paving the way for the new presets along the way
3. The _"VHS Tapes"_ category has been renamed to _"Tapes"_

4. Effects added:
  - Low-power/far analog effect by @Degamisu (#151)
  - @notdigidani's camcorder preset (#114)
  - A 1993 Worn Tape by @Degamisu (#155)
  - An EMTEC E-240 Tape effect by @VexorionReal (#142)
  - An Analog Drone Effect by @nlolnlolnlolnlo (#136)
  - Somewhat Damaged 2004 Tape by @ThrivertonVHS (#149)
  - An Open Reel VHS Tape by @Degamisu (#156 / Suggested Topic: #140)

5. From now on: changelogs are now moved to "CHANGELOG.txt"
6. The **"NTSC-RT Ultimate Preset Pack"** are now moved to the Discussions tab.

Here's the file: 
[NTSC-RT Ultimate Preset Pack V0.1.1.zip](https://github.com/user-attachments/files/16563465/NTSC-RT.Ultimate.Preset.Pack.V0.1.1.zip)

**Comment by notdigidani** [comment]
love this

**Reply by smfthe1199** [reply]
Thanks

**Comment by smfthe1199** [comment]
I should mention that the v0.1.0 changelog has been edited to look professional, and I have edited the README.txt file to edit to make it for the new version.

**Reply by smfthe1199** [reply]
New version available: https://github.com/valadaptive/ntsc-rs/discussions/171
